config/initializers/paper_trail.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+PaperTrail.config.enabled = true
+PaperTrail.config.has_paper_trail_defaults = { on: %i[update] }
+PaperTrail.config.version_limit = 3
```

Comment:
Let's increase this to 10
```suggestion
PaperTrail.config.version_limit = 10
```

Reply:
Ok, increased 🫡.[Commit hash](https://github.com/espoo-dev/med_system_backend/pull/175/commits/9cbcb622e99045b6ade4c0c3fe2240736e3798b5) `9cbcb622e99045b6ade4c0c3fe2240736e3798b5`